file-format-types.rdf

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+  xmlns:owl="http://www.w3.org/2002/07/owl#"
+  xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+  xmlns:udfrs="http://www.udfr.org/onto#">
+  <owl:Ontology rdf:about="http://pcdm.org/file-format-types#">
+    <rdfs:comment xml:lang="en">PCDM File Format Genre ontology.</rdfs:comment>
+    <rdfs:label xml:lang="en">PCDM File Format Genres</rdfs:label>
+  </owl:Ontology>
+  <udfrs:GenreFacetType rdf:ID="Document">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <skos:exactMatch rdf:resource="http://vocab.getty.edu/aat/300026031" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#Document" />
+    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#DocumentGenre" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Archive">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Document" />
+    <skos:exactMatch rdf:resource="http://vocab.getty.edu/aat/300375748" />
+    <skos:exactMatch rdf:resource="http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#Archive" />
+    <skos:closeMatch rdf:resource="http://reference.data.gov.uk/technical-registry/Aggregate" />
+    <skos:closeMatch rdf:resource="http://www.udfr.org/onto#AggregateGenre" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Database">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Document" />
+    <skos:exactMatch rdf:resource="http://vocab.getty.edu/aat/300028543" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/Database" />
+    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#DatabaseGenre" />
+  </udfrs:GenreFacetType>
+  <udfrs:GenreFacetType rdf:ID="Dataset">
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/file-format-types#"/>
+    <rdfs:subClassOf rdf:resource="Document" />
+    <skos:exactMatch rdf:resource="http://reference.data.gov.uk/technical-registry/Dataset" />
+    <skos:exactMatch rdf:resource="http://www.udfr.org/onto#DatasetGenre" />
+    <skos:exactMatch rdf:resource="http://id.loc.gov/vocabulary/resourceTypes/Dat" />
+    <skos:exactMatch rdf:resource="http://purl.org/dc/dcmitype/Dataset" />
+  </udfrs:GenreFacetType>
```

Comment:
See http://test.linkeddatapronom.nationalarchives.gov.uk/vocabulary/pronom-vocabulary.htm -- the basic difference is that a database is used with a DBMS (e.g., relational database, MS Access, FileMaker Pro, etc.), and dataset is more likely to be text, such as CSV or tabular formats, export from SAS or other packages, etc.  Database here could be a subclass of Dataset.